### DISCUSSION TITLE
Save all the image from the page

### DISCUSSION THREAD

**Discussion by codescooper** [kevinzg/facebook-scraper#699] [General]
Hello thank for this script, 
The work that i have to acheive is take all the picture of our page by scraping and save into a folder. 
all i have see till now is get the url of the image, 
can you tell me how is possible to save the corresponding image localy ? 

Sorry for my sad english ^^

**Comment by neon-ninja** [comment]
You can use `requests` for this. Here's an example:

```python
import requests
from facebook_scraper import *
from urllib.parse import urlparse
import os

os.makedirs("images", exist_ok=True)
for post in get_posts("Nintendo"):
    for url in post["images"]:
        filename = os.path.basename(urlparse(url).path)
        print(f"Fetching {filename}")
        r = requests.get(url)
        with open(f"images/{filename}", "wb") as f:
            f.write(r.content)
```
